web/controllers/respondent_controller.ex

```diff
@@ -606,6 +606,7 @@ defmodule Ask.RespondentController do
     # Now traverse each respondent and create a row for it
     csv_rows = respondents
     |> Stream.map(fn respondent ->
+
```

Comment:
Is this whitespace required by the linter? Let's not add spurious whitespace, otherwise :)